src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml

```diff
@@ -11,6 +11,7 @@
     <link type="image/x-icon" rel="shortcut icon" href="/TheTheme/favicon.ico" />
     <script asp-name="bootstrap" use-cdn="true" version="4" at="Foot"></script>
     <style asp-name="bootstrap" use-cdn="true" version="4"></style>
+    <style asp-src="/TheTheme/css/sticky-footer-navbar.css"></style>
```

Comment:
Why do we need that? Isn't Bootstrap 4 able to render a simple theme by itself?

Reply:
Not if the navbar is fixed.